app/components/content/row.js

```diff
@@ -3,7 +3,7 @@ import styled, { css } from 'styled-components'
 import { colors } from '@libscie/design-library'
 import subtypes from '@hypergraph-xyz/wikidata-identifiers'
 import { useHistory, useLocation } from 'react-router-dom'
-import Plus from './plus.svg'
+import childContentButton from './child-content-button.svg'
```

Comment:
```suggestion
import ChildContentButton from './child-content-button.svg'
```

React components always start upper case